qcengine/util.py

```diff
@@ -436,28 +459,32 @@ def disk_files(infiles: Dict[str, str], outfiles: Dict[str, None], cwd: Optional
         lwd = Path.cwd()
     else:
         lwd = Path(cwd)
+    if as_binary is None:
+        as_binary = []
```

Comment:
Should we do any kinds of checks to make sure the binary values exist in the outfiles?

Reply:
"binary values exist"? any file can be opened as binary, so the usual file exists check should be sufficient, unless I'm misunderstanding.

Reply:
More like: `outfiles = {"hello"}, as_binary={"nonexistent"}`

Could prevent some confusion if we do something like:
 `assert len(set(as_binary) - set(outfiles)) == 0`

Reply:
`as_binary` list is going to overlap with both `infiles` and `outfiles` dictionaries, so the existing OS errors should still catch. you're worried about `as_binary` never getting iterated over, so a mismatch could happen? I suspect this is a programmer's option, not an end-point option, so errors will get caught pretty quick. otherwise, I guess could pop as consumed and raise an error at the end if any remain.

Reply:
so asserted.